tests/circuit_parser_test.py

```diff
@@ -1,9 +1,11 @@
+import numpy as np
 from qulacs import QuantumCircuit
+from qulacs.gate import CNOT, SWAP, TOFFOLI, DenseMatrix, to_matrix_gate
 
 from qulacsvis.visualization.circuit_parser import CircuitParser
 
 
-def test_matplotlib_init() -> None:
+def circuit_parser_test1() -> None:
```

Comment:
関数名は`test_`で始まるようにしないとpytestでの実行対象にならないです。例えば、以下のようにすると良いです。

```suggestion
def test_circuit_parser_example1() -> None:
```